modules/javafx.graphics/src/main/java/javafx/stage/Window.java

```diff
@@ -657,10 +682,6 @@ void notifySizeChanged(double newWidth, double newHeight) {
 
     /**
      * Whether or not this {@code Window} has the keyboard or input focus.
-     * <p>
```

Comment:
Is this removal intentional?

Reply:
Yes, this is intentional. This property has no setter, so the existing comment is in error.